File: nodejs-api/server/model/account.ts
import mongoose from 'mongoose';
const { Schema, model } = mongoose;

const schema = new Schema({
    email: {
        type: String,
    },
    name: {
        type: String,
    },
    language: {
        type: String,
    },
    created: {
        type: Date,
    },
    modified: {
        type: Date,
    }
});

const Account = model("Account", schema);

export default Account;

File: springboot/src/main/java/nl/elstarit/event/service/model/Account.java
package nl.elstarit.event.service.model;

import com.fasterxml.jackson.annotation.JsonInclude;
import lombok.Data;
import org.springframework.data.annotation.CreatedDate;
import org.springframework.data.annotation.Id;
import org.springframework.data.annotation.LastModifiedDate;
import org.springframework.data.mongodb.core.mapping.Document;

import java.time.LocalDateTime;

@Data
@Document
@JsonInclude(JsonInclude.Include.NON_NULL)
public class Account {

    @Id
    private String id;
    private String name;
    private String language;
    private String email;

    @CreatedDate
    private LocalDateTime created;

    @LastModifiedDate
    private LocalDateTime modified;
}
